README.md

```diff
@@ -1,5 +1,9 @@
 # gistup
 Simple CLI tool to upload file to GIST using Github API
+
+## requirements
+requests
```

Comment:
Since this is a simple CLI, I think it's feel to keep it third-party depedencies free. Let's try to use the standard library that works on both Python 2 and Python 3.

Reply:
That seems reasonable! I have made the changes requested.